book/07-git-tools/sections/credentials.asc

```diff
@@ -30,20 +30,20 @@ You can choose one of these methods by setting a Git configuration value:
 $ git config --global credential.helper cache
 ----
 
-Some of these helpers have options.
-The ``store'' helper can take a `--file <path>` argument, which customizes where the plain-text file is saved (the default is `~/.git-credentials`).
-The ``cache'' helper accepts the `--timeout <seconds>` option, which changes the amount of time its daemon is kept running (the default is ``900'', or 15 minutes).
-Here's an example of how you'd configure the ``store'' helper with a custom file name:
+Marami sa mga tumutulong ay mayroong mga opsyon.
+Ang ``store'' na katulong ay maaaring kumuha ng isang `--file <path>` na argumento, na kung saan ay which nagpapasadya sa kung saan ang plain-teksto na file ay naka-save (ang default ay `~/.git-credentials`).
+Ang ``cache'' na katulong ay tumatanggap ng `--timeout <seconds>` na opsyon, na kung saan ay nagbabago sa halaga ng oras sa kanyang daemon ay pinanatiling tumatakbo (ang default ay ``900'', o 15 minuto).
+Ito ang isang halimbawa kung paano mo siya i-configure ang ``store'' na katulong na may isang pasadyang pangalan ng file:
```

Comment:
change paano mo siya i-configure to paano mo ikompigura

Reply:
done change